### PR TITLE
Update Changelog 5.49.7

### PR DESCRIPTION
**Pull request by NewKidOnTheBlock**
Thank you for your contribution to the Sandboxie repository.

In order to reduce the risk of accidental merges, it's highly recommended for beginners to open a new Pull Request in draft state by clicking the button "Create Draft Pull Request", instead of using the default "Create Pull Request". You can convert an existing pull request to a draft by clicking the "Convert to draft" link in the right sidebar under "Reviewers".

### CODE REVIEW

**Review by isaak654 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,43 +16,43 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
+- align default settings of AutoRecover and Favourites to the Plus version (thanks isaak654)
+- list of email clients and browsers is now centralized in Dll_GetImageType
 
 ### Fixed
 - fixed minor issue with logging internet blocks
 - fixed issue with file recovery when located on a network share
-- fixed ui issue with CallTrace
-- fixed crated sandbox links gettign double extension
+- fixed UI issue with CallTrace
+- fixed sandbox links receiving double extension
```
Comment (isaak654) [inline CHANGELOG.md:34]: - fixed the creation of new sandbox links receiving double extension

It's related to #770

**Review by isaak654 — CHANGES_REQUESTED** [review]
Mmm, maybe you just need to improve this:

File: CHANGELOG.md (hunk 1 above)
Comment (NewKidOnTheBlock) [reply CHANGELOG.md:34]: Hmm. I don't know how to edit a pull request short of deleting the whole thing and creating a new one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,43 +16,43 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
 ### Changed
 - reworked window hooking mechanism to improve performance
--- resolves issues with file save dialogs taking 30+ sec to open
+-- resolves issues with file save dialogues taking 30+ seconds to open
 -- this fix greatly improves the win32 GUI performance of sandboxed processes
-- reworked RPC resolver to be ini configurable
+- reworked RPC resolver to be ini-configurable
 -- the following options are now deprecated:
 --- "UseRpcMgmtSetComTimeout=some.dll,n", so use "RpcPortBinding=some.dll,*,TimeOut=y"
 --- "OpenUPnP=y", "OpenBluetooth=y", "OpenSmartCard=n" use the new templates instead
```
Comment (isaak654) [inline CHANGELOG.md:54]: Suggestion:
--- "OpenUPnP=y", "OpenBluetooth=y", "OpenSmartCard=n", so use the new RPC templates instead

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (NewKidOnTheBlock) [reply CHANGELOG.md:54]: This completely changes the meaning (not to claim that I have understood line 54):

Before
"(This), (this) and (this) all use the new template."

After
"(This), (this) and (this) have been removed, so you should use the new template."

Comment (isaak654) [reply CHANGELOG.md]: I think the second is more appropriate, for example there is a lot of disabled code with double slashes if you search OpenBluetooth in the source: https://github.com/sandboxie-plus/Sandboxie/search?q=OpenBluetooth

Besides, the new templates write different values:
```
Template=SSDP
Template=OpenSmartCard
Template=OpenBluetooth
```

**Review by isaak654 — APPROVED** [review]
I think you can remove the draft status from the pull request.

**Pull request merged**